evap/results/tests/test_views.py

```diff
@@ -699,3 +701,35 @@ def test_archived_results(self):
         self.app.get(url, user='manager', status=200)
         self.app.get(url, user='reviewer', status=403)
         self.app.get(url, user='student_external', status=403)
+
+
+class TestTextAnswerExportView(WebTest):
+
+    @classmethod
+    def setUpTestData(cls):
+        baker.make(UserProfile, username="reviewer", groups=[Group.objects.get(name="Reviewer")])
+        evaluation = baker.make(Evaluation)
+
+        baker.make(TextAnswer, _quantity=3, __contribution__evaluation=evaluation)
+        cls.url = f"/results/evaluation/{evaluation.id}/text_answers_export"
+
+    def test_file_sent(self):
+        def mock(_self, res):
+            res.write(b"1337")
+
+        with patch.object(TextAnswerExcelExporter, "export", mock):
+            response = self.app.get(self.url, user="reviewer", status=200)
+            self.assertEqual(response.headers["Content-Type"], "application/vnd.ms-excel")
+            self.assertEqual(response.content, b"1337")
+
+    @patch("evap.results.exporters.TextAnswerExcelExporter.export")
+    def test_permission_denied(self, export_method):
+        baker.make(UserProfile, username="student")
+        baker.make(UserProfile, username="manager", groups=[Group.objects.get(name="Manager")])
+
+        self.app.get(self.url, user="student", status=403)
+        export_method.assert_not_called()
+        self.app.get(self.url, user="reviewer", status=200)
+        export_method.assert_called_once()
+        self.app.get(self.url, user="manager", status=200)
+        self.assertEqual(export_method.call_count, 2)
```

Comment:
I think this is fine here - just as a future reference: The mocks have a `reset_mock` method, allowing you to reset the recorded calls.